model.go

```diff
@@ -343,3 +343,13 @@ type MediaUpload struct {
 	Image            Image  `json:"image"`
 	Video            Video  `json:"video"`
 }
+
+// MediaInsights represents a Twitter's response object for a call to their insights API.
+type MediaInsights struct {
+	Favourites  string `json:"favorites"`
+	Replies     string `json:"replies"`
+	Retweets    string `json:"retweets"`
+	VideoViews  string `json:"expires_after_secs"`
```

Comment:
VideoViews != expires_after_seconds ?

Reply:
Where did that come from??? Changed it. Thank you!